app/views/foods/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Here are your results!</h1>
+
+<p>Total Results: <%= @results.count %> </p>
```

Comment:
`@results` has been limited to 10, meaning this will never return the true total count of search results.